doc/user/unit-testing.md

```diff
@@ -254,4 +254,4 @@ func TestMemcachedController(t *testing.T) {
 [doc-reconcile]: https://godoc.org/sigs.k8s.io/controller-runtime/pkg/reconcile#Reconciler
 [code-test-example]: https://github.com/operator-framework/operator-sdk-samples/blob/master/memcached-operator/pkg/controller/memcached/memcached_controller_test.go#L25
 [user-guide]: ../user-guide.md#register-with-the-managers-scheme
-[ocp-doc-v1-route]: https://docs.openshift.com/container-platform/3.11/rest_api/apis-route.openshift.io/v1.Route.html
\ No newline at end of file
+[ocp-doc-v1-route]: https://docs.openshift.com/container-platform/3.11/rest_api/apis-route.openshift.io/v1.Route.html
```

Comment:
What changed here, I can't see a difference 🤔

Reply:
Humm let me checkout that, I just one to change the cl vs client